server/main.lua

```diff
@@ -28,6 +28,23 @@ CreateThread(function()
     end
 end)
 
+local validatePosition = function(coords)
```

Comment:
```suggestion
local function validatePosition(coords)
```
This the syntax we use everywhere